packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/participants/local-participant-view.mdx

```diff
@@ -3,8 +3,42 @@ id: local-participant-view
 title: LocalParticipantView
 ---
 
+LocalParticipantView displays the local video stream of the local participant if the video is unmuted. It can be used as both floating or full-screen view.
```

Comment:
```suggestion
LocalParticipantView displays the local video stream of the local participant if the video is unmuted. It can be used as both a floating or full-screen view.
```